app/common/_temp_hooks/useAmountLogic.tsx

```diff
@@ -61,7 +61,7 @@ export const useAmountLogic = ({ selectedAsset, isGift }: AmountLogicParams) =>
   }
 
   const getIsAccountToBeReaped = (): boolean => {
-    if (!touched || transferAll || !maxAmountToSend || !fee) return false;
+    if (!amount || !parseFloat(amount) || !touched || transferAll || !maxAmountToSend || !fee) return false;
```

Comment:
Will you use BN instead of string manipulations in #155 ?

Reply:
No, I've created a dedicated issue in [clickup](https://app.clickup.com/t/869508mvw) to move to BN.